support/metrics/metricsTracker.go

```diff
@@ -286,3 +350,36 @@ func (mt *MetricsTracker) sendEvent(eventType string, eventProps interface{}) er
 	}
 	return nil
 }
+
+func mergeEventProps(commonProps, eventProps map[string]interface{}) (map[string]interface{}, error) {
+	m := make(map[string]interface{})
```

Comment:
any benefits to using `make` instead of `map[string]interface{}{}`?

Reply:
I think that throws a segfault upon the initial assignment, since the memory is not allocated.